source/multigrid/mg_tools.cc

```diff
@@ -1511,8 +1511,9 @@ namespace MGTools
 
     for (; cell!=endc; ++cell)
       {
+        // do not look at artificial cells
         if (mg_dof_handler.get_tria().locally_owned_subdomain()!=numbers::invalid_subdomain_id
```

Comment:
If you run in serial, see https://www.dealii.org/developer/doxygen/deal.II/classTriangulation.html#a18c743385f69a7125942d0d03d8b1e13

(I am updating the comment)